app/models/user.rb

```diff
@@ -4,4 +4,9 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   has_one :cart
+
+  def admin?
```

Comment:
What is the purpose of this method ?
I believe its for checking the role.

Try to compare with constant not the hard coded value.